recipes/default.rb

```diff
@@ -18,6 +18,17 @@
 #
 include_recipe 'firewall::http'
 
-node['osl-nginx']['recipes'].each do |recipe|
-  include_recipe "chef_nginx::#{recipe}"
+nginx_install 'repo'
+
+# make this availible for notify statements
+service 'nginx' do
+  action :nothing
+end
+
+# TODO: Let's remove CentOS 6 as soon as we can
+if node['platform']['version'].to_i < 7
+  package 'nginx' do
+    notifies :restart, 'service[nginx]'
+    action [:remove, :install]
```

Comment:
This is going to happen on every converge and should have some kind of guard. Is there something you can check to know when this needs to happen?

Although, at this point I think it might better to just remove EL6 support since we only have one host left. Let's do that instead.

Reply:
sounds good